### ISSUE TITLE
Create MysqlCluster Failed

### ISSUE THREAD

**Issue by luoyangustc** [bitpoke/mysql-operator#614]
When I use mysql-operator/charts/mysql-cluster to generate the mysql-cluster.yaml,The content show as follow:
```yaml
apiVersion: mysql.presslabs.org/v1alpha1
kind: MysqlCluster
metadata:
  name: mysql-cluster-db
  labels:
    app: mysql-cluster
    chart: mysql-cluster-0.3.0
    release: mysql-cluster
    heritage: Tiller
spec:
  replicas: 3
  secretName: mysql-cluster-db
  volumeSpec:
    hostPath:
      path: /tmp
      type: DirectoryOrCreate
```

The result :
![image](https://user-images.githubusercontent.com/31932405/96414355-96e7ec80-121f-11eb-9ee0-d2e4b12e3c12.png)
The pod of mysql-cluster-db-mysql-0 is Ok,but mysql-cluster-db-mysql-1 is Error.
kubectl logs mysql-cluster-db-mysql-1 -c init:
![image](https://user-images.githubusercontent.com/31932405/96414515-ce569900-121f-11eb-9b2b-fc8bee2e916f.png)

what should i do !I need Help so much!
Thanks a lot!

**Comment by AMecea** [comment]
Hi @luoyangustc, can you please check on which node did the second pod scheduled? 

Please make sure about host path directory permissions. The mysql user `999` should have permissions over that directory on the node.

Also, using `hostPath` it will require some node selector and affinity in place to keep the pod on the same node. Please try running, at first, only with one a node and set node selector on the cluster to make sure that the pod is always scheduled on the same node.

Please let me know if you found a solution to this! Best regards,

**Comment by munjalpatel** [comment]
@luoyangustc were you ever able to solve this? I am having the same exact issue (https://github.com/presslabs/mysql-operator/issues/653). Only major difference between our setup is that I am using a PVC.

**Comment by calind** [comment]
Just released v0.5.0. Check release notes at https://github.com/bitpoke/mysql-operator/releases/tag/v0.5.0.

Please reopen if issue still persists.
